src/test/resources/fixtures/hmh/hmh-ri/development/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,49 @@
+{
+	"@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+	"type": "AssessmentEvent",
+	"id": "urn:uuid:27734504-068d-4596-861c-2315be33a2a2",
+	"actor": {
+		"id": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+		"type": "Person"
+	},
+	"action": "Started",
+	"object": {
+		"id": "urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+		"type": "Assessment",
+		"name": "ReadingInventory",
+		"maxAttempts": 1,
+		"maxSubmits": 1,
+		"maxScore": 25.0,
+		"version": "1.0"
+	},
+	"eventTime": "2016-11-15T10:15:00.000Z",
+	"edApp": {
```

Comment:
Ed Analytics expects edApp to be a string - having it as a json block will cause the pipeline to block this event and divert it to the 'invalid events' sink and consequently, this event won't flow through the pipeline.